src/lib/IFrameCommunicatorTypes.ts

```diff
@@ -74,9 +74,9 @@ export enum AddButtonElementType {
 }
 
 export enum AddButtonPermission {
-  CreateNew = 'CreateNew',
-  Edit = 'Edit',
-  ViewParent = 'ViewParent',
+  CreateNew = 'createNew',
+  Edit = 'edit',
+  ViewParent = 'viewParent',
```

Comment:
It surprised me a bit that these permission names are not starting with a capital. It seems like the BE returns the permission model already in this state. Do you think it makes sense to update the permission contract on the client too? The current types work well, though, so it is probably not a real issue.

Reply:
@milan009 according to contract [here](https://kentico.atlassian.net/wiki/spaces/KPM/pages/2697068925/KCL-6719+Spike+notes+-+revised) those permission names should start with capital letter. Could you please explain why it needs to be changed?

Reply:
Property names used in the response sent from BE are using lowerCamelCase - this may be a result of us using some sort of a global JSON resolver in our BE solution. I forgot to make this change on client side as well, and it honestly kinda baffles me that it works - I guess we have case conversion handled somehow at the client side? My suggestion would be to update our contract definition to use lowerCamelCase and replicate these changes in the client solution as well. 

![image](https://user-images.githubusercontent.com/9932636/118081651-1277d980-b3bc-11eb-9443-33c199505e1c.png)